src/robotFactory.js

```diff
@@ -1,10 +1,63 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, { x = 0, y = 0 }, chipVersion) {
+    this.name = name;
+    this.weight = weight;
 
-class FlyingRobot {}
+    this.coords = {
+      x,
+      y,
+    };
+    this.chipVersion = chipVersion;
+  };
+  getInfo() {
+    // eslint-disable-next-line max-len
+    return `Robot: ${this.name}, Chip version: ${this.chipVersion}, Weight: ${this.weight}`;
+  }
+  goForward(step = 1) {
+    this.coords.y += step;
+  };
+  goBack(step = 1) {
+    this.coords.y -= step;
+  };
+  goRight(step = 1) {
+    this.coords.x += step;
+  };
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  };
```

Comment:
Let's make some breaklines between methods
```suggestion
  getInfo() {
    // eslint-disable-next-line max-len
    return `Robot: ${this.name}, Chip version: ${this.chipVersion}, Weight: ${this.weight}`;
  }
  
  goForward(step = 1) {
    this.coords.y += step;
  };
  
  goBack(step = 1) {
    this.coords.y -= step;
  };
  goRight(step = 1) {
    this.coords.x += step;
  };
  
  goLeft(step = 1) {
    this.coords.x -= step;
  };
```